colcon_bundle/installer/base_pip_installer.py

```diff
@@ -84,7 +84,7 @@ def install(self):  # noqa: D102
         python_pip_args = [self._python_path, '-m', 'pip']
         pip_install_args = python_pip_args + ['install']
         subprocess.check_call(
-            pip_install_args + ['-U', 'pip', 'setuptools==44.0.0'])
+            pip_install_args + ['-U', 'pip==20.*', 'setuptools==44.0.0'])
```

Comment:
Should this have a check to see if it is on Xenial first? On other versions it should be fine to not lock?

Reply:
Alternative comment - does this then attempt to upgrade to latest? The intermediate upgrade to 20 is intended so that pip then understands the newer version metadata that describes compatibility. We want to have newer versions where it is supported